packages/autocomplete-js/src/__tests__/panelContainer.test.ts

```diff
@@ -1,3 +1,65 @@
+import { waitFor } from '@testing-library/dom';
+
+import { autocomplete } from '../autocomplete';
+
 describe('panelContainer', () => {
-  test.todo('tests');
+  let container;
+  let panelContainer;
+
+  beforeEach(() => {
```

Comment:
Can we inline this in the tests? It's harder to read like this and some test setup is not used everywhere.